src/components/molecules/NavBar/NavBar.scss

```diff
@@ -270,3 +270,16 @@ $border-radius: 28px;
     display: none;
   }
 }
+
+.nav-sparkle-back-home-icon {
```

Comment:
It's plausible, since this whole file is legacy, but we have a bit different patterns for class names.

If we were to follow it, it would be something like `.NavBar__home-icon`

Reply:
fixed